aspnetcore/fundamentals/error-handling.md

```diff
@@ -77,70 +57,57 @@ public IActionResult Error()
 
 ## Configure status code pages
 
-By default, an app doesn't provide a rich status code page for HTTP status codes, such as *404 Not Found*. To provide status code pages, use Status Code Pages Middleware.
-
-::: moniker range=">= aspnetcore-2.1"
+By default, an app doesn't provide a rich status code page for HTTP status codes, such as *404 - Not Found*. To provide status code pages, use Status Code Pages Middleware.
 
 The middleware is made available by the [Microsoft.AspNetCore.Diagnostics](https://www.nuget.org/packages/Microsoft.AspNetCore.Diagnostics/) package, which is available in the [Microsoft.AspNetCore.App metapackage](xref:fundamentals/metapackage-app).
 
-::: moniker-end
-
-::: moniker range="= aspnetcore-2.0"
-
-The middleware is made available by the [Microsoft.AspNetCore.Diagnostics](https://www.nuget.org/packages/Microsoft.AspNetCore.Diagnostics/) package, which is available in the [Microsoft.AspNetCore.All metapackage](xref:fundamentals/metapackage).
-
-::: moniker-end
-
-::: moniker range="< aspnetcore-2.0"
-
-The middleware is made available by adding a package reference for the [Microsoft.AspNetCore.Diagnostics](https://www.nuget.org/packages/Microsoft.AspNetCore.Diagnostics/) package in the project file.
-
-::: moniker-end
-
 Add a line to the `Startup.Configure` method:
 
 ```csharp
 app.UseStatusCodePages();
 ```
 
-<xref:Microsoft.AspNetCore.Builder.StatusCodePagesExtensions.UseStatusCodePages*> should be called before request handling middlewares in the pipeline (for example, Static File Middleware and MVC Middleware).
+Call the <xref:Microsoft.AspNetCore.Builder.StatusCodePagesExtensions.UseStatusCodePages*> method before request handling middleware (for example, Static File Middleware and MVC Middleware).
 
-By default, Status Code Pages Middleware adds text-only handlers for common status codes, such as 404:
+By default, Status Code Pages Middleware adds text-only handlers for common status codes, such as *404 - Not Found*:
```

Comment:
This answers my earlier question -- I would put this at the start of the section.